hack/build/builder/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+#Copyright 2023 The AAQ Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+set -e
+
+script_dir="$(cd "$(dirname "$0")" && pwd -P)"
+
+source "${script_dir}"/common.sh
+
+source "${script_dir}"/../common.sh
+
+VERSION=$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)
+if [ "${1}" != "" ]; then
+    VERSION="${1}"
+fi
```

Comment:
i think we can just return the VERSION here and avoid the input stuff similar to [how it is being done in kubevirt](https://github.com/kubevirt/kubevirt/blob/904b5b49c339b1a3cbdc430c71da89d6353f4286/hack/builder/publish.sh#L24)

Reply:
I would honestly like to find a different way, since the downside of outputting it to stdout as done in kubevirt is that you have to redirect all other command outputs to stderr.
I would rather have the version explicitly set by the caller instead. Or maybe i can just source the build script instead? Let me experiment here a bit.

Reply:
I see what you mean.